common/src/main/java/com/google/auto/common/MoreElements.java

```diff
@@ -17,20 +17,29 @@
 package com.google.auto.common;
 
 import com.google.common.annotations.Beta;
+import com.google.common.base.Function;
 import com.google.common.base.Optional;
+import com.google.common.base.Preconditions;
+import com.google.common.base.Predicate;
+import com.google.common.collect.FluentIterable;
+import com.google.common.collect.ImmutableList;
+
 import java.lang.annotation.Annotation;
+
+import static javax.lang.model.element.ElementKind.PACKAGE;
```

Comment:
BTW, `ElementKind` is already imported on its own, so that static import is a bit redundant.